ENV.MD

```diff
@@ -0,0 +1,427 @@
+
+##### LDAP_QUERY_FILTER_USER
+
+  - e.g. `(&(mail=%s)(mailEnabled=TRUE))`
+  - => Specify how ldap should be asked for users
+
+##### LDAP_QUERY_FILTER_GROUP
+
+  - e.g. `(&(mailGroupMember=%s)(mailEnabled=TRUE))`
+  - => Specify how ldap should be asked for groups
+
+##### LDAP_QUERY_FILTER_ALIAS
+
+  - e.g. `(&(mailAlias=%s)(mailEnabled=TRUE))`
+  - => Specify how ldap should be asked for aliases
+
+##### LDAP_QUERY_FILTER_DOMAIN
+
+- e.g. `(&(|(mail=*@%s)(mailalias=*@%s)(mailGroupMember=*@%s))(mailEnabled=TRUE))`
+- => Specify how ldap should be asked for domains
+
+##### DOVECOT_TLS
+
+  - **empty** => no
+  - yes => LDAP over TLS enabled for Dovecot
+
+## Dovecot
+
+The following variables overwrite the default values for ```/etc/dovecot/dovecot-ldap.conf.ext```.
+
+##### DOVECOT_USER_FILTER
+
+  - e.g. `(&(objectClass=PostfixBookMailAccount)(uniqueIdentifier=%n))`
+
+##### DOVECOT_USER_ATTRS
+
+ - e.g. `homeDirectory=home,qmailUID=uid,qmailGID=gid,mailMessageStore=mail`
+ - => Specify the directory to dovecot attribute mapping that fits your directory structure.
+ - Note: This is necessary for directories that do not use the [Postfix Book Schema](test/docker-openldap/bootstrap/schema/mmc/postfix-book.schema).
+ - Note: The left-hand value is the directory attribute, the right hand value is the dovecot variable.
+ - More details on the [Dovecot Wiki](https://wiki.dovecot.org/AuthDatabase/LDAP/Userdb)
+
+##### DOVECOT_PASS_FILTER
+
+  - e.g. `(&(objectClass=PostfixBookMailAccount)(uniqueIdentifier=%n))`
+
+##### DOVECOT_PASS_ATTRS
+
+- e.g. `uid=user,userPassword=password`
+- => Specify the directory to dovecot variable mapping that fits your directory structure.
+- Note: This is necessary for directories that do not use the [Postfix Book Schema](test/docker-openldap/bootstrap/schema/mmc/postfix-book.schema).
+- Note: The left-hand value is the directory attribute, the right hand value is the dovecot variable.
+- More details on the [Dovecot Wiki](https://wiki.dovecot.org/AuthDatabase/LDAP/PasswordLookups)
+
+## Postgrey
+
+##### ENABLE_POSTGREY
+
+  - **0** => `postgrey` is disabled
+  - 1 => `postgrey` is enabled
+
+##### POSTGREY_DELAY
+
+  - **300** => greylist for N seconds
+
+Note: This postgrey setting needs `ENABLE_POSTGREY=1`
+
+##### POSTGREY_MAX_AGE
+
+  - **35** => delete entries older than N days since the last time that they have been seen
+
+Note: This postgrey setting needs `ENABLE_POSTGREY=1`
+
+##### POSTGREY_AUTO_WHITELIST_CLIENTS
+
+  - **5** => whitelist host after N successful deliveries (N=0 to disable whitelisting)
+
+Note: This postgrey setting needs `ENABLE_POSTGREY=1`
+
+##### POSTGREY_TEXT
+
+  - **Delayed by postgrey** => response when a mail is greylisted
+
+Note: This postgrey setting needs `ENABLE_POSTGREY=1`
+
+## SASL Auth
+
+##### ENABLE_SASLAUTHD
+
+  - **0** => `saslauthd` is disabled
+  - 1 => `saslauthd` is enabled
+
+##### SASLAUTHD_MECHANISMS
+
+  - empty => pam
```

Comment:
instead of empty list pam, make it as default and explain pam (its the only unexplained at the moment)